app/models/upvs/message_draft.rb

```diff
@@ -51,13 +51,14 @@ def self.load_from_params(message_params, box:)
     message = ::Message.create(message_params.except(:objects, :tags).merge(
         {
           sender_name: box.name,
-          # recipient_name: TODO search name in UPVS dataset,
           outbox: true,
           replyable: false,
           delivered_at: Time.now
         }
       )
     )
+    message.update(recipient_name: Upvs::ServiceWithFormAllowRule.matching_metadata(message).where(institution_uri: message.metadata['recipient_uri'])&.take&.institution_name)
```

Comment:
Nevieme to spravit v jednom kroku, ze to posunieme do toho merge tam vyssie?

Reply:
Chcela som, ale robilo by sa to tazsie, pokym este nie je sprava vytvorena. Potrebujeme metadata spravy na volanie uz existujucej metody `matching_metadata`

Reply:
@luciajanikova nedava zmysel potom hore nevolat create ale build, nastavit vsetko ako treba a potom `save!` ?

Reply:
updatela som @jsuchal